npm/next/7.0/@n4jsd/next/lib/dist/server/next.n4jsd

```diff
@@ -1,7 +1,7 @@
 import {IncomingMessage} from "http";
 import {ServerResponse} from "http";
 import {Socket} from "net";
-import {INextRouter} from "next/router";
+import {INextRouter} from "lib/router";
```

Comment:
lib?

Reply:
IDE resolves INextRouter incorrectly as "router" for `app.n4jsd`. This is why we have an outer lib folder.